src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -583,5 +618,79 @@ private string[] GetCompatibilityLevels(int sqlServerVersion, DatabasePrototype
             // previous loop did not find the prototype compatibility level in this server's compatability options, so treat compatibility levels as unsupported for this server
             return Array.Empty<string>();
         }
+
+        /// <summary>
+        /// Get supported backup redundancy levels for this Azure server.
+        /// </summary>
+        private static string[] GetAzureBackupLevels()
+        {
+            return AzureSqlDbHelper.BackupStorageRedundancyLevels;
+        }
+
+        /// <summary>
+        /// Get supported service level objectives for this Azure server.
+        /// </summary>
+        private static AzureEditionDetails[] GetAzureServiceLevels(string[] editionDisplayNames)
+        {
+            var levels = new List<AzureEditionDetails>();
+            foreach (string displayName in editionDisplayNames)
+            {
+                if (AzureSqlDbHelper.TryGetAzureEditionFromDisplayName(displayName, out var azureEdition))
+                {
+                    if (AzureSqlDbHelper.TryGetServiceObjectiveInfo(azureEdition, out var serviceInfoPair))
+                    {
+                        // Move default value to the front of the list
+                        var serviceLevelsList = new List<string>(serviceInfoPair.Value);
+                        var defaultIndex = serviceInfoPair.Key;
+                        if (defaultIndex >= 0 && defaultIndex < serviceLevelsList.Count)
+                        {
+                            var defaultServiceObjective = serviceLevelsList[defaultIndex];
+                            serviceLevelsList.RemoveAt(defaultIndex);
+                            serviceLevelsList.Insert(0, defaultServiceObjective);
+                        }
+                        var details = new AzureEditionDetails() { EditionDisplayName = displayName, Details = serviceLevelsList.ToArray() };
+                        levels.Add(details);
+                    }
+                }
+            }
+            return levels.ToArray();
+        }
+
+        /// <summary>
+        /// Get supported editions for this Azure server.
+        /// </summary>
+        private static string[] GetAzureEditions()
```

Comment:
`GetAzureEditionDisplayNames`? (and update doc comment too)

Reply:
Removed this method since it was just passing a single property back